RESULTS.md

```diff
@@ -570,5 +590,94 @@ examples from the first 400 moves should mostly rid us of those pointless
 examples without dramatically distorting the ratio of opening/middle/endgame
 examples seen by the network.
 
+This run ended up not really having fixed anything. On our cross-evaluation 
+matches, v12 ended up slightly stronger than the other runs, but it was well
+within the natural variability of the other runs. V12 still had ladder issues
+and it still could not beat LeelaZero or ELF, even with increased playouts.
+
+### v14, Bigtable integration with init-to-loss switch
+
+For v14, the biggest change was replacing the ad-hoc in-memory shuffler with
+a BigTable shuffler.
+
+Previously, we'd write out tiny TFRecords containing ~200
+TFExamples each, and another Python process would rsync the records to one
+machine, read them into memory, and maintain a large moving window of positions
+in memory that would be periodically be spit out as training chunks. This shuffler
+had all sorts of hacks in it - we'd entirely ignore any games with < 30 moves;
+we'd ignore any moves with move number > 400; we'd draw a fixed number (4) of
+moves from each game (which oversampled short games). This hacky fixed-number
+sampling was because exactly 10% of our games were resign-disabled, but those
+resign disabled games were much longer than the other games, meaning that an
+even sampling by move would oversample our resign-disabled games. A fixed-number
+sampling would at least guarantee a 10% representation of resign-disabled games.
+
+In terms of bottlenecks, our sampler took a big chunk of time to run, but it
+was still shorter than the training time, so it was not technically a bottleneck
+yet. But we were still training on a single TPU (8 cores) rather than a pod
+(128 cores), so if we were to scale to more TPUs for selfplay, and upgrade our
+trainer to use a pod to compensate, then this shuffler would quickly become the
+bottleneck.
+
+For v14, our selfplay workers would instead directly upload each move to BigTable
+as a (position, next_move, winner) tuple. A set of parallel readers would then
```

Comment:
The TFExamples stored in the Bigtable row were identical to the TFExamples that were stored in the previous approach. Each TFExample was stored in the Bigtable with a row key based on its game index and move index. Each TFExample contains, I believe, 8 moves of history, just as before.